packages/dids/.mocharc.json

```diff
@@ -1,5 +1,9 @@
 {
   "enable-source-maps": true,
   "exit": true,
-  "spec": ["tests/compiled/**/*.spec.js"]
-}
\ No newline at end of file
+  "spec": ["tests/compiled/**/*.spec.js"],
+  "node-option": [
+    "experimental-specifier-resolution=node",
+    "loader=ts-node/esm"
+  ]
```

Comment:
any specific reason for using `ts-node`? we've gotten by without having to use it so far.

Reply:
it lets me run mocha tests in webstorm really simply

happy to remove it and just re-add locally / remove before i push if easier

Reply:
ah gotcha. did you have to configure something specific to get things to work with webstorm? i'm thinking we can get webstorm to continue working really simply without introducing `ts-node`. 

p.s. we should move the `ts-node` dependency to `devDependencies` in `package.json`

Reply:
yeah had to manually add a run config, happy to work with you to find another solution